corehq/apps/reports/views.py

```diff
@@ -1937,20 +1937,26 @@ def archive_form(request, domain, instance_id):
 
 
 def _get_cases_with_forms_message(domain, cases_with_other_forms, case_id_from_request):
-    def _get_case_link(case_id, name):
-        if case_id == case_id_from_request:
-            return _("%(case_name)s (this case)") % {'case_name': name}
-        else:
-            return '<a href="{}#!history">{}</a>'.format(reverse('case_data', args=[domain, case_id]), name)
+    def _get_all_case_links():
+        all_case_links = []
+        for case_id, case_name in cases_with_other_forms.items():
+            if case_id == case_id_from_request:
+                all_case_links.append(
+                    format_html(_("{} (this case)"), case_name)
+                )
+            else:
+                all_case_links.append(format_html(_(
+                    '<a href="{}#!history">{}</a>'),
+                    reverse("case_data", args=[domain, case_id]),
+                    case_name
+                ))
```

Comment:
Nit: it looks like the previous version did not translate this string, and it sort of makes sense why -- `<a>case_name</a>` won't have any translatable strings, apart from `case_name`. If we want to stick with translating this, I'd recommend just limiting the translation to `_(case_name)`

Reply:
Makes sense 👍

Reply:
I am planning to remove the translation from this link because I don't think we translate inputs provided by the user.